openff/interchange/_tests/interoperability_tests/test_openmm.py

```diff
@@ -455,6 +455,7 @@ def test_nocutoff_when_nonperiodic(self, de_force_field):
             ):
                 assert force.getNonbondedMethod() == openmm.NonbondedForce.NoCutoff
 
+    @pytest.mark.skip(reason="Different energies with different upstreams")
```

Comment:
(not blocking) Would be good to document which ones

Reply:
Will document more publicly in #1190